### ISSUE TITLE
Lax @nuxt/kit requirement version

### ISSUE THREAD

**Issue by IlyaSemenov** [cpsoinos/nuxt-svgo#140]
The module currently wants `@nuxt/kit-edge@latest` as a runtime dependency:

https://github.com/cpsoinos/nuxt-svgo/blob/b30c65b5e3d034a1cb198bf351355de34d02ac92/package.json#L48

Is there a specific reason to do so? This leads to duplicate `@nuxt/kit` pulled for no apparent reason.

Will the module not work with `"@nuxt/kit": "^3.4.0"`?

**Comment by jd1378** [comment]
I am not aware of the reason
@cpsoinos is there any specific reason ?

**Comment by IlyaSemenov** [comment]
I quickly tested with `package.json` overrides in my project:

```json
{
  "pnpm": {
    "overrides": {
      "@nuxt/kit": "~3.6.2"
    }
  }
}
```

and it worked just fine, both dev and prod build. I think it should also work with `^3.4.0` so if possible please lax it to that semver (I am stuck at Nuxt 3.4 at some other project due to multiple suspense issues in 3.5+).

**Comment by jd1378** [comment]
Okay, I'll downgrade and run tests, if it works without any problem, sure

**Comment by jd1378** [comment]
It seems this is the way https://github.com/nuxt/module-builder recommends setting it up
I tried to use `^3.4.0` but latest module-builder requires at least `3.5.0`

**Comment by jd1378** [comment]
resolved by #141
